src/backend/joanie/tests/payment/test_api_credit_card.py

```diff
@@ -75,7 +75,7 @@ def test_api_credit_card_get_credit_cards_list(self):
         self.assertEqual(response.status_code, 200)
         content = response.json()
         results = content.pop("results")
-        self.assertEqual(
+        self.assertCountEqual(
```

Comment:
I think it's better to keep the check on the ids in the list and not just the length.
We can order the cards'list from the factory to align with the new ordering and keep the `assertEqual`

```suggestion
        cards.sort(key=lambda x: x.created_on, reverse=True)
        self.assertEqual(
```